xExtension-ReadingTime/static/readingtime.js

```diff
@@ -42,18 +42,20 @@
 
  flux_words_count: function flux_words_count(flux) {
 
-     reading_time.textContent = flux.textContent; // get textContent
+     reading_time.textContent = flux.querySelector('.flux_content').textContent; // get textContent, from the article itself (not the header, not the bottom line). `childNodes[2].childNodes[1]` gives : `<div class="content no_limit">` element
```

Comment:
You should use `'.flux_content .content'` to be sure to get only the content to read